cmdLine/src/main/scala/topgun/cmdline/records/ClassInfo.java

```diff
@@ -0,0 +1,55 @@
+package topgun.cmdline.records;
+
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.tree.ClassNode;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ClassInfo {
```

Comment:
this needs to be less lazy. 
Please read the information at construction time and populate a case class